pkg/internal/checkup/checkup.go

```diff
@@ -259,6 +265,32 @@ func (c *Checkup) createTrafficGeneratorPod(ctx context.Context) error {
 	return nil
 }
 
+func (c *Checkup) getPCIDevicesVarName(ctx context.Context) (string, error) {
+	netAttachDef, err := c.client.GetNetworkAttachmentDefinition(ctx, c.namespace, c.params.NetworkAttachmentDefinitionName)
+	if err != nil {
+		return "", err
+	}
+
+	const resourceNameAnnotationKey = "k8s.v1.cni.cncf.io/resourceName"
+	rawResourceName, exists := netAttachDef.Annotations[resourceNameAnnotationKey]
+	if !exists {
+		return "", fmt.Errorf("%q annotation is not found on the network-attachement-definition %q/%q",
+			resourceNameAnnotationKey, c.namespace, c.params.NetworkAttachmentDefinitionName)
+	}
+
+	const (
+		pciDevicesVarPrefix    = "PCIDEVICE"
+		pciDevicesVarDelimiter = "_"
+	)
+
+	// build the PCI Device env var naming recipe
+	// https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin#pod-device-information
+	pciDevicesVarName := pciDevicesVarPrefix + pciDevicesVarDelimiter + rawResourceName
```

Comment:
The variable name should also be upper-cased:
https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/blob/5f8442e5b92f15e5f63d6352efbf1b2e1108b574/pkg/resources/pool_stub.go#L118

Reply:
yeah I saw it as soon as I ran it.. Fixed